test/testAll.m

```diff
@@ -44,6 +44,9 @@
 
 if ~isempty(strfind(getenv('HOME'), 'jenkins')) || ~isempty(strfind(getenv('USERPROFILE'), 'jenkins'))
     WAITBAR_TYPE = 0;
+
+    % add CellNetAnalyzer for testing purposes
+    addpath(genpath(getenv('CNA_PATH')));
```

Comment:
You would need a checkCNAInstallation here as well.
Otherwise libSBML usage depends on the order of the tests.

Reply:
👍